rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -349,7 +350,7 @@ static void ppu_initialize_modules()
 	}
 
 	// Set memory protection to read-only
-	vm::page_protect(ppu_function_manager::addr, ::align(::size32(hle_funcs) * 8, 0x1000), 0, 0, vm::page_writable);
+	utils::memory_protect(vm::base(ppu_function_manager::addr), ::size32(hle_funcs) * 8, utils::protection::ro);
```

Comment:
It's not a new API for this purpose, it's just different. Memory under the vm management stores its own page information.